src/services/infra/DynamoDBService.ts

```diff
@@ -0,0 +1,72 @@
+import autoBind from "auto-bind"
+
+import { config } from "@config/config"
+
+import {
+  MessageBody,
+  SiteLaunchLambdaStatus,
+} from "@root/../microservices/site-launch/shared/types"
+
+import DynamoDBClient, { UpdateParams } from "./DynamoDBClient"
+
+export default class DynamoDBService {
+  private readonly dynamoDBClient: DynamoDBClient
+
+  private readonly TABLE_NAME: string
+
+  // todo: delete these mock after integration
+  private mockLaunch: MessageBody = {
```

Comment:
nit: prefer shifting this outside and not exporting + rename to `SCREAMING_SNAKE_CASE`

Reply:
super small nit: can we put as `TODO:`
Vscode tools like https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight help highlight these annotations like TODO and FIXME

Reply:
@harishv7 823d1059

Reply:
@seaerchin done! 9c38a9